### PR TITLE
Rasised memory limit in the dev image to 256M

### LINKED ISSUE #107: [BUG] Allowed memory size of 134217728 bytes exhausted
**Describe the bug**
docker exec kimai2 /opt/kimai/bin/console kimai:reset-dev exceeds the memory limit

**To Reproduce**
Steps to reproduce the behaviour:
1. ```docker run --rm -ti -p 8001:8001 --name kimai2 --env APP_ENV=dev kimai/kimai2:apache-debian-1.8-dev```
2. ```docker exec kimai2 /opt/kimai/bin/console kimai:reset-dev```
4. See error ```Allowed memory size of 134217728 bytes exhausted```

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04

**Additional context**
We just need to up the memory limit in the dev image

### PR DESCRIPTION
**Pull request by tobybatch**
Fixes #107

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -229,7 +229,8 @@ RUN export COMPOSER_HOME=/composer && \
     composer install --working-dir=/opt/kimai --optimize-autoloader && \
     composer clearcache && \
     composer require --working-dir=/opt/kimai laminas/laminas-ldap && \
-    chown -R www-data:www-data /opt/kimai
+    chown -R www-data:www-data /opt/kimai && \
+    sed "s/128M/256M/g" /usr/local/etc/php/php.ini-development > /usr/local/etc/php/php.ini
```
Comment (kevinpapst) [inline Dockerfile:233]: I would feel safer if you replace the complete line `memory_limit=128M` with `memory_limit=256M`.

Is command line and fpm php using the same php.ini?

**Comment by Schrolli91** [comment]
Why is kimai needed as much RAM? Is that a normal behavior?

**Comment by kevinpapst** [comment]
This is not runtime memory, but RAM when pre-populating the database with test data. Due to the nature of the data structure tenth of thousands of objects are kept in RAM while the fixtures are imported.

**Comment by Schrolli91** [comment]
ah ok - now i understand the problem - thanks

File: Dockerfile (hunk 1 above)
Comment (tobybatch) [reply Dockerfile:233]: There is only one php.ini in the fpm image

**Review by kevinpapst — APPROVED** [review]

**Review by Schrolli91 — APPROVED** [review]
LGTM

**Pull request merged**